scripts/env/ci.sh

```
@@ -0,0 +1,1091 @@
+    log_info "Action: $action"
+    
+    # Example of the recommended pattern:
+    # docker run --rm -it \
+    #   -v /var/run/docker.sock:/var/run/docker.sock \
+    #   -v "$(pwd)":/workspace \
+    #   -w /workspace \
+    #   docker:24.0.2 \
+    #   sh -c "docker compose $compose_files $action"
+    
+    # For direct usage (when already in container with mounted socket):
+    docker compose $compose_files $action
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)